wallet/concepts/wallet-interoperabilty.md

```diff
@@ -0,0 +1,41 @@
+---
+description: Learn about wallet interoperability.
+sidebar_position: 6
+---
+
+# Wallet interoperability
+
+A web dapp can integrate with multiple wallets simultaneously by adding support for
+[EIP-6963](https://eips.ethereum.org/EIPS/eip-6963), which improves the user experience when the
+user has multiple wallet browser extensions installed.
+EIP-6963 introduces an alternative discovery mechanism to the [`window.ethereum` injected provider](apis.md#ethereum-provider-api),
+enabling dapps to discover multiple injected wallet providers in a user's browser.
+
+<p align="center">
+  <video width="100%" controls>
+    <source src="/eip-6963/eip-6963.mov" />
+  </video>
+</p>
+
+You can add support for EIP-6963 in one of the following ways:
+
+- [Set up MetaMask SDK](../how-to/connect/set-up-sdk/javascript/index.md) in your dapp.
+  The SDK supports EIP-6963 by default, and we recommend using this method.
+- [Directly update your dapp code](../how-to/discover-multiple-wallets.md) to support EIP-6963.
```

Comment:
i feel like a lot of what is said on this linked page is the same as what is already on this page itself. Would it make sense to have the discover-multiple-wallets page simply link back this page instead?

Reply:
The `discover-multiple-wallets` page does link back to this page (the first sentence links to "wallet interoperability"). It does contain a lot of the same intro content, but it's more of placeholder for when we have more thorough instructions for directly updating the dapp code.